<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>turbulence</class>
 <widget class="QScrollArea" name="turbulence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>298</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="apply_button">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/newPrefix/images/fromHelyx/save16.png</normaloff>:/newPrefix/images/fromHelyx/save16.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_turbulence">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Turbulence</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioTurb1">
         <property name="text">
          <string>Laminar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioTurb2">
         <property name="text">
          <string>RANS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioTurb3">
         <property name="text">
          <string>LES</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxTurb">
         <item>
          <property name="text">
           <string>Select Model</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioTurb1</sender>
   <signal>toggled(bool)</signal>
   <receiver>turbulence</receiver>
   <slot>changeTurbList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioTurb2</sender>
   <signal>toggled(bool)</signal>
   <receiver>turbulence</receiver>
   <slot>changeTurbList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioTurb3</sender>
   <signal>toggled(bool)</signal>
   <receiver>turbulence</receiver>
   <slot>changeTurbList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apply_button</sender>
   <signal>pressed()</signal>
   <receiver>turbulence</receiver>
   <slot>aplicar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTurb</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>turbulence</receiver>
   <slot>changeComboBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeTurbList()</slot>
  <slot>aplicar()</slot>
  <slot>changeComboBox()</slot>
 </slots>
</ui>
